### PR TITLE
Add import of totp from Lastpass

### PR DESCRIPTION
**Pull request by djsmith85**
Fixes bitwarden/desktop#795

The lastpassCsvImporter wasn't copying the totp value into the cipher.

Added test to verify

### CODE REVIEW

File: spec/common/importers/lastpassCsvImporter.spec.ts [hunk 1]
```diff
@@ -1,10 +1,29 @@
 import { LastPassCsvImporter as Importer } from '../../../src/importers/lastpassCsvImporter';
 
+import { ImportResult } from '../../../src/models/domain/importResult';
 import { CipherView } from '../../../src/models/view/cipherView';
 import { FieldView } from '../../../src/models/view/fieldView';
 
+import { CipherType } from '../../../src/enums';
 import { FieldType } from '../../../src/enums';
```
Comment (cscharf) [inline spec/common/importers/lastpassCsvImporter.spec.ts:8]: Perhaps you can consolidate the imports from `../../../src/enums` as:
```typescript
import { CipherType, FieldType } from '../../../src/enums';
```

**Review by cscharf — COMMENTED** [review]
Looks great, thanks @djsmith85 ! Just one small convention item on the imports, while in the test class probably not a big deal, but would be good to follow established patterns.

File: spec/common/importers/lastpassCsvImporter.spec.ts (hunk 1 above)
Comment (djsmith85) [reply spec/common/importers/lastpassCsvImporter.spec.ts]: Thank you @cscharf for the review.  I've fixed the import as requested.

**Review by cscharf — APPROVED** [review]

**Comment by cscharf** [comment]
@djsmith85 , thanks again for the fix! Any chance you're up to pulling this jslib update into desktop?

**Comment by djsmith85** [comment]
@cscharf Sure, no problem 😎

**Pull request merged**